src/tools/compiletest/src/header.rs

```diff
@@ -1135,6 +1135,8 @@ fn parse_normalize_rule(header: &str) -> Option<(String, String)> {
     .captures(header)?;
     let regex = captures["regex"].to_owned();
     let replacement = captures["replacement"].to_owned();
+    // FIXME: Support escaped new-line in strings.
+    let replacement = replacement.replace("\\n", "\n");
```

Comment:
Discussion: so I'm not *super* convinced by adding the logic immediately here -- won't this affect other unrelated tests?

Reply:
It will affect every `normalize-` header, but since none of those currently have a `\n` in them, no other tests will be affected (other than the one I'm adding).

There is currently no way to a new-line in the replacement string, I tried a bunch of things and just gave up and added this mini logic.